src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/DockerProcessor.kt

```diff
@@ -17,27 +17,32 @@ package com.bardsoftware.backend.fleet.rmarkdown
 
 import com.spotify.docker.client.DefaultDockerClient
 import com.spotify.docker.client.DockerClient
+import com.spotify.docker.client.DockerClient.LogsParam.stderr
+import com.spotify.docker.client.DockerClient.LogsParam.stdout
 import com.spotify.docker.client.messages.ContainerConfig
 
 class DockerProcessor {
     private val docker: DockerClient = DefaultDockerClient.fromEnv().build()
 
-    fun getMd5Sum(message: String): String? {
+    fun getMd5Sum(message: String): String {
         var containerId: String? = null
 
         try {
             val quotedMessage = "\"$message\""
 
             val containerConfig = ContainerConfig.builder()
                     .image("busybox")
-                    .cmd("sh", "-c", "md5sum", "<<<", quotedMessage)
+                    .cmd("sh", "-c", "echo $quotedMessage | md5sum")
                     .build()
 
             val creation = docker.createContainer(containerConfig)
             containerId = creation.id()
             docker.startContainer(containerId)
 
-            return docker.logs(containerId, DockerClient.LogsParam.stdout()).readFully()
+            docker.logs(containerId, stdout(), stderr()).use({
```

Comment:
Please add `return` statement here, even if it is optional. Otherwise it does look as if you're always throwing an exception.

```
            return docker.logs(containerId, stdout(), stderr()).use({
                stream -> stream.readFully()
            }) ?: ""
```

Reply:
Done